deny.toml

```diff
@@ -214,9 +214,9 @@ allow-registry = ["https://github.com/rust-lang/crates.io-index"]
 # List of URLs for allowed Git repositories
 allow-git = []
 
-#[sources.allow-org]
+[sources.allow-org]
 # 1 or more github.com organizations to allow git sources for
-#github = [""]
+github = ["TheDan64"]
```

Comment:
nit: Comment for why this is allows would be good.

I already wondered with other settings in `deny.toml.`

Reply:
This is not needed anymore, we are using regular version of inkwell now